composer.json

```diff
@@ -57,6 +57,14 @@
                 "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch"
             }
         },
+        "patches-ignore": {
```

Comment:
Hi,
Why do we have this ignore section in this commit? 
Is this related to the body class?

Reply:
The issue with the patches across components will be solved in a follow up ticket, so for now I'm ok with merging this PR.